test/logstream/v2/stream.go

```diff
@@ -34,21 +34,25 @@ import (
 	"knative.dev/pkg/ptr"
 )
 
-func FromNamespaces(ctx context.Context, c kubernetes.Interface, namespaces []string) Source {
+func FromNamespaces(ctx context.Context, c kubernetes.Interface, namespaces []string, filterLines bool, podPrefixes ...string) Source {
```

Comment:
I don't have a good suggestion on what to do here that's not a breaking change.

Reply:
This is addressed in the latest commit.